### DISCUSSION TITLE
User Denied Media Access even though all permissions are granted

### DISCUSSION THREAD

**Discussion by MarkusPint** [versatica/JsSIP#881] [Q&A]
Hi,
I have seen several cases where I have verified that the user has given:
1) OS-level permissions to the browser for camera and microphone access
2) Browser-level permissions for the website for camera and microphone access

And user still gets the error "User denied media access".

In one case the user says this happens intermittently, sometimes fixed by a restart (OS X).

Has anyone else seen things like that and any pointers to debug it further would be appreciated.

**Comment by expositor** [comment]
Hi, were you able to resolve this issue?
